lib/src/client/options.dart

```diff
@@ -21,11 +21,11 @@ import 'dart:math';
 import '../shared/security.dart';
 
 const defaultIdleTimeout = const Duration(minutes: 5);
+const defaultConnectTimeout = const Duration(seconds: 20);
```

Comment:
I think we should not give a default connection timeout - that deviates from Socket.connect, and is therefore surprising.